package-name-map/README.md

```diff
@@ -0,0 +1,6 @@
+# package-name-map
```

Comment:
I'd prefer the folder be called reference-implementation (and the h1 be "Reference implementation")

Reply:
If this is published as an npm package, we'd want a descriptive name, like `package-name-map`.